sonic-slave-bullseye/Dockerfile.j2

```diff
@@ -501,6 +501,11 @@ RUN patch -p1 -i /disable-non-manylinux.patch /usr/local/lib/python3.9/dist-pack
 # For building sonic-utilities
 RUN pip3 install fastentrypoints mock
 
+# For building sonic_ycabled
+# Note: upstream build breaks with old version of setuptools
+# ref: https://github.com/grpc/grpc/issues/34569
+RUN pip3 install grpcio==1.58.0 grpcio-tools==1.58.0
```

Comment:
Looks like the dockers and the base image are on GRPC 1.39.0. Do we want to use that version instead?

Reply:
If the current situation is truly any version, it is using relative newer version until recent build broken. So I prefer not to change situation dramatically.